e2etest/newe2e_resource_managers_local.go

```diff
@@ -297,6 +297,13 @@ func (l *LocalObjectResourceManager) ListChildren(a Asserter, recursive bool) ma
 func (l *LocalObjectResourceManager) GetProperties(a Asserter) ObjectProperties {
 	out := ObjectProperties{}
 
+	stats, err := os.Stat(l.getWorkingPath())
+	a.NoError("failed to get stat", err)
+	lmt := common.Iff(stats == nil, nil, PtrOf(stats.ModTime()))
+	out.FileProperties = FileProperties{
```

Comment:
OK yeah, this is poorly named. FileProperties was designed for Files-service specific properties, not just any file, sorry for the confusion 😅

Reply:
moving this to ObjectProperties